pkg/config/manager_test.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"reflect"
+	"testing"
+)
+
+// TestConfig_DefaultConfig asserts the default configuration attributes is in use when a
+// non-existing configuration file is informed.
+func TestConfig_DefaultConfig(t *testing.T) {
```

Comment:
sorry forgot to note @otaviof that the second unit test you added best as I can see address things I noted per https://github.com/openshift/csi-driver-shared-resource/pull/60#discussion_r732178310